Python/Product/PythonTools/PythonTools/Options/PythonAdvancedEditorOptionsControl.Designer.cs

```diff
@@ -171,6 +173,12 @@ partial class PythonAdvancedEditorOptionsControl {
             this.tableLayoutPanel1.Controls.Add(this._miscOptionsGroupBox, 0, 2);
             this.tableLayoutPanel1.Name = "tableLayoutPanel1";
             // 
+            // contextMenuStrip1
```

Comment:
what is this change for?

Reply:
I added and then removed some UI from this control, which will have modified the auto-generated code. Haven't manually reverted this one to the old revision yet.